test/widget/src/server/linked_in_auth_code_widget_test.dart

```diff
@@ -115,4 +115,37 @@ void main() {
 
     expect(find.text('AppBar title'), findsOneWidget);
   });
+
+  testWidgets('has PreferredSizeWidget type, if App Bar is provided', (
```

Comment:
there is already test which is testing almost same thing `App bar is shown if it is provided`

Reply:
what I meant by adding a test is to create a dummy custom AppBar widget which you are able to provide to testable widget and find by newly created custom type widget. (Sorry for misunderstanding I was typing from phone)